### PR TITLE
[RAINCATCH-1258] UI guide

### PR DESCRIPTION
**Pull request by paolobueno**
- [X] Mention technologies: Angular, Bootstrap
- [X] Describe how projects are formatter.
- [x] How people can add new elements to UI. 
- [x] How people can modify styles.
- [x] Adding their own categories to navigator.

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-angularjs-modules.adoc
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-angularjs-modules.adoc]: need to rebase master {WFM-RC-NameShort} is now {Feature}

File: docs/workforce-management-framework/topics/pro-ui-customization.adoc
Comment (austincunningham) [inline docs/workforce-management-framework/topics/pro-ui-customization.adoc]: WIP

File: docs/workforce-management-framework/topics/ref-demo-app.adoc
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-demo-app.adoc]: getting rid of the discrete now we have a dynamic toc

File: docs/workforce-management-framework/topics/ref-demo-app.adoc
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-demo-app.adoc]: Issue with the table render 

image

**Comment by wtrocki** [comment]
@paolobueno Great work so far. Everything makes sense. I will wait for the rest to do final review

File: docs/workforce-management-framework/topics/ref-demo-app.adoc
Comment (austincunningham) [reply docs/workforce-management-framework/topics/ref-demo-app.adoc]: needs an extra =
```|===```

File: docs/workforce-management-framework/topics/con-ui-technology.adoc
Comment (wtrocki) [inline docs/workforce-management-framework/topics/con-ui-technology.adoc]: Users may not need to know where they are kept- We have separate contributor guide that we going to use for this.

Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-ui-technology.adoc]: Just lets avoid confusion from end users checking this repo.

Comment (paolobueno) [reply docs/workforce-management-framework/topics/con-ui-technology.adoc]: Great, I wasn't sure about this part, any links required?

Going to reword to just mention the framework itself.

Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-ui-technology.adoc]: No links required as people will get applications depending on their support level upstream etc.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
@jstaffor Ready for action!

**Comment by paolobueno** [comment]
@wtrocki changed target to `upstream`, could be okay to merge now :pray:

**Pull request merged**
